api/messaging/spec/webhooks_public.json

```diff
@@ -0,0 +1,75 @@
+{
+  "Title": "Webhooks (Public)",
+  "Interface": "Webhooks_public",
+  "Struct": null,
+  "Parameters": {},
+  "Protocol": "",
+  "Authentication": [],
+  "Path": "/webhooks",
+  "APIs": [
+    {
+      "Name": "webhookDelete",
+      "Method": "DELETE",
+      "Title": "Delete webhook",
+      "Path": "/{webhookID}/{webhookToken}",
```

Comment:
It resolves a possible security issue where a webhook may be exposed, and a user with permissions to delete it might be unavailable - but the webhook URL is accessible. 

> - the reasoning behing this one is more out of convenience. If some webhook link would leak and would be discovered outside of the sysadmin teams, they could invalidate the webhook just by issuing a DELETE statement. Implementation at this point: optional, but easy.

Documented as motivation in #101